### PR TITLE
[Zip][Portable] Add Ngrok.Ngrok 3.1.1

### PR DESCRIPTION
**Pull request by rodrigoborgesdeoliveira**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/96427)

### CODE REVIEW

File: manifests/n/Ngrok/Ngrok/3.1.1/Ngrok.Ngrok.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# PublisherSupportUrl:
+PrivacyUrl: https://ngrok.com/privacy
+Author: ngrok, Inc.
+PackageName: Ngrok
+PackageUrl: https://ngrok.com
+License: Proprietary
+# LicenseUrl:
+# Copyright:
+# CopyrightUrl:
+ShortDescription: Tunnel local ports to public URLs and inspect traffic
```
Comment (mdanish-kh) [inline manifests/n/Ngrok/Ngrok/3.1.1/Ngrok.Ngrok.locale.en-US.yaml:18]:
```suggestion
ShortDescription: ngrok is a simplified API-first ingress-as-a-service that adds connectivity,security, and observability to your apps with no code changes
```
I would suggest to stick with the description on the PackageUrl https://ngrok.com

File: manifests/n/Ngrok/Ngrok/3.1.1/Ngrok.Ngrok.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+PackageLocale: en-US
+Publisher: ngrok, Inc.
+PublisherUrl: https://ngrok.com
+# PublisherSupportUrl:
+PrivacyUrl: https://ngrok.com/privacy
+Author: ngrok, Inc.
+PackageName: Ngrok
+PackageUrl: https://ngrok.com
+License: Proprietary
+# LicenseUrl:
```
Comment (mdanish-kh) [inline manifests/n/Ngrok/Ngrok/3.1.1/Ngrok.Ngrok.locale.en-US.yaml:15]:
```suggestion
LicenseUrl: https://ngrok.com/tos
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

File: manifests/n/Ngrok/Ngrok/3.1.1/Ngrok.Ngrok.locale.en-US.yaml (hunk 1 above)
Comment (rodrigoborgesdeoliveira) [reply manifests/n/Ngrok/Ngrok/3.1.1/Ngrok.Ngrok.locale.en-US.yaml:18]: I used the descriptions returned by `ngrok -h`.
image

Thoughts?

File: manifests/n/Ngrok/Ngrok/3.1.1/Ngrok.Ngrok.locale.en-US.yaml (hunk 2 above)
Comment (rodrigoborgesdeoliveira) [reply manifests/n/Ngrok/Ngrok/3.1.1/Ngrok.Ngrok.locale.en-US.yaml]: Note for the future: ngrok has a [copyright link](https://ngrok.com/copyright) as pointed [here](https://ngrok.com/tos#:~:text=https%3A//ngrok.com/copyright) that is currently down. If it ever comes back online, we can use it in the manifest.

image

**Review by mdanish-kh — APPROVED** [review]

File: manifests/n/Ngrok/Ngrok/3.1.1/Ngrok.Ngrok.locale.en-US.yaml (hunk 1 above)
Comment (mdanish-kh) [reply manifests/n/Ngrok/Ngrok/3.1.1/Ngrok.Ngrok.locale.en-US.yaml:18]: Ah right, I was wondering where you got that description from. Both work for me as they come directly from the publisher.

**Pull request merged**